Enable C++ mode in the C compiler the standard way.
Do not override the MacPorts deployment target and stdlib choices.
Fixes build with non-clang compilers, which don't understand -stdlib.
https://github.com/wbolster/plyvel/pull/91
--- setup.py.orig	2018-01-16 12:49:11.000000000 -0600
+++ setup.py	2018-11-08 12:53:25.000000000 -0600
@@ -14,9 +14,7 @@
         return fp.read()
 
 
-extra_compile_args = ['-Wall', '-g']
-if platform.system() == 'Darwin':
-    extra_compile_args += ['-mmacosx-version-min=10.7', '-stdlib=libc++']
+extra_compile_args = ['-Wall', '-g', '-x', 'c++']
 
 ext_modules = [
     Extension(
